linkerd/manifest.json

```diff
@@ -28,6 +28,8 @@
       "Linkerd - Overview": "assets/dashboards/overview.json"
     },
     "service_checks": "assets/service_checks.json",
-    "logs": {}
+    "logs": {
+      "source": "linkerd"
```

Comment:
I think we can also add `log collection` to `manifest.categories` for Linkerd.

Reply:
Done in https://github.com/DataDog/integrations-core/pull/7523